### ISSUE TITLE
Aptitude will not allow 7 skill points for Exotic Weapon Skills

### ISSUE THREAD

**Issue by chummer5a** [chummer5a/chummer5a#108]
<a href="https://github.com/GoogleCodeExporter"><img src="https://avatars.githubusercontent.com/u/9614759?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [GoogleCodeExporter](https://github.com/GoogleCodeExporter)**
_Tuesday Mar 17, 2015 at 08:56 GMT_
_Originally opened as https://github.com/chummer5a/chummer5/issues/34_

----

```
What steps will reproduce the problem?
1. Grant Exotic Weapon Skill (ranged or melee).
2. Grant Aptitude (Exotic Weapon Skill (ranged or melee))
3. Try to increase skill to 7

What is the expected output? What do you see instead?
Should have a skill at 7 points

What version of the product are you using? On what operating system?
Version 0.0.5.139 on Windows 7

Please provide any additional information below.


```

Original issue reported on code.google.com by `exte...@gmail.com` on 13 Jan 2015 at 5:33

**Comment by chummer5a** [comment]
Technicallly not currently an issue. Skills running over the limit aren't validated. Need to run some error checking against that.

**Comment by chummer5a** [comment]
It's always reassuring when I come across broken code that I didn't write.

Issue stems from a flaw in the improvement manager thinking that the Exotic Weapon descriptor is a specialisation. Basically just need some extra logic to check whether specialisations would actually be descriptors.

**Comment by chummer5a** [comment]
Issue resolved in 5.170.
